src/components/ChatLog.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+// import TimeStamp from './TimeStamp';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({ entries, updateLike }) => {
+  const chatComponents = entries.map((message) => {
+    return (
+      <div key={message.id} className="chat-entry local">
```

Comment:
Nice job having a key for each entry! 

Make sure you aren't assigning unused classes though. There aren't any CSS rules for `chat-entry` or `local` in the ChatLog.css file. This may have been left here on accident, but I wanted to call it out just in case.